### ISSUE TITLE
Question: Is there an option to use only polling (get Pilot) rather than the heartbeats/sync pilot? 

### ISSUE THREAD

**Issue by SRGDamia1** [sbidy/pywizlight#162]
I have 15 running on a network with home assistant. Skimming the code, it looks like the bulbs are sending requests every 30 seconds to keep the 5s heart beats coming. It's it possible to reduce network traffic by using reduced polling?

I spent a lot of time developing the openHab Wiz integration, but I finally got sick of my inability to make dashboards friendly enough for my family with openhab and switched to home assistant.

**Comment by treaki** [comment]
what, a couple small UDP datagrams every 30 seconds times 15 are to much, what are you using, wifi 1 (eg. 802.11-1999) with 11mbits from the 90s?

if you want to use a dashboard you should use some OOP/MVC to abstract and cache your values and dont ask for a different network protocol i guess

**Comment by sbidy** [comment]
No, you could adjust the lib by yourself but in general as @treaki mentioned this should not be an big issue.
